dub.json

```diff
@@ -16,8 +16,12 @@
 	"lflags-posix": [
 		"-L$PACKAGE_DIR/libxlsxwriter/build/"
 	],
+	"lflags-windows-x86_mscoff-dmd": [
+		"\\\"/LIBPATH:$PACKAGE_DIR\\install_dir\\libxlsxwriter\\win32\\lib\\\"",
+		"\\\"/LIBPATH:$PACKAGE_DIR\\install_dir\\zlib\\win32\\lib\\\""
+	],
 	"lflags-windows-x86_64-dmd": [
-		"\\\"/LIBPATH:$PACKAGE_DIR\\install_dir\\libxlsxwriter\\lib\\x64\\Release\\\"",
+		"\\\"/LIBPATH:$PACKAGE_DIR\\install_dir\\libxlsxwriter\\lib\\\"",
```

Comment:
shouldn't this point to 

"\"\"/LIBPATH:$PACKAGE_DIR\\install_dir\\libxlsxwriter\\lib\\x64\\Release\"\"",

similar to the ldc libpath?

Reply:
> shouldn't this point to
> 
> """/LIBPATH:$PACKAGE_DIR\install_dir\libxlsxwriter\lib\x64\Release""",
> 
> similar to the ldc libpath?

Really.
I got the latest versions of libraries from the git repository, this is an error.